content/doc/developer/views/exposing-bundled-resources.adoc

```diff
@@ -0,0 +1,94 @@
+---
+title: Exposing bundled resources
+layout: developerguide
+---
+
+Resource files bundled with Jenkins and plugins are by default exposed through several different mechanisms.
+
+## Static resource files
+
+Jenkins exposes static resource files of allowed file types (`.js`, `.css`, `.jpeg`, `.jpg`, `.png`, `.gif`, `.html`, `htm` as of Jenkins 2.290) in core and plugins at the `/resources/` URL.
```

Comment:
```suggestion
Jenkins exposes static resource files of allowed file types (`.js`, `.css`, `.jpeg`, `.jpg`, `.png`, `.gif`, `.html`, `.htm` as of Jenkins 2.290) in core and plugins at the `/resources/` URL.
```

Reply:
Jenkins 2.290 is the next weekly release.  I don't see a merged pull request that is adding that capability.  Is 2.290 the correct version number?

I see mention in the [2.204.1 changelog](https://www.jenkins.io/changelog-stable/#v2.204.1) of a change related to static resources but don't see anything more recent than that

Reply:
@MarkEWaite I am making the educated guess that there will not be changes. I could have written 2.289 (or 1.164 for that matter). The list never changed.

Reply:
I misunderstood your intent.  The phrase "as of 2.290" is stating that the list was extracted from Jenkins 2.290, not that there was a change made in 2.290 to enable that feature.  Thanks for the clarification!